docs_app/package.json

```diff
@@ -23,10 +23,10 @@
     "test-pwa-score": "node scripts/test-pwa-score",
     "deploy-production": "scripts/deploy-to-firebase.sh",
     "payload-size": "scripts/payload.sh",
-    "docs": "dgeni ./tools/transforms/angular.io-package && npm run docs-decision-tree",
+    "docs": "npx ts-node -P tsconfig.docs.json ./node_modules/dgeni/lib/gen-docs.js ./tools/transforms/angular.io-package && npm run docs-decision-tree",
     "docs-watch": "node tools/transforms/authors-package/watchr.js",
```

Comment:
I now finally noticed what was missing for me. I used the 'serve-and-sync' task which is using the 'docs-watch' command. This isn't compiling the ts code for now. Could you maybe add this here to make it fully working.

Reply:
@JWO719 I finally fixed the `docs-watch` script.